### DISCUSSION TITLE
Wierd loud sound after entering a beatmap.

### DISCUSSION THREAD

**Discussion by Zexarr** [ppy/osu#25912] [Q&A] [answered]
I am getting somewhat of a loud sound now after entering a beatmap and it's irritating. I didn't noticed it before I think. Also this happened on an Android device. It's the Nothing Phone (1).

https://github.com/ppy/osu/assets/152501410/76ae10dd-64c8-4786-b1b1-f7bd8accc929

Don't know if logs are needed but do tell me if needed.

**Comment by KoleckOLP** [comment]
I have that as well on some beatmaps but not all, I'm on Windows.

**Comment by bdach** [comment, marked as answer]
already tracked as https://github.com/ppy/osu/issues/25875
